### DISCUSSION TITLE
好好好

### DISCUSSION THREAD

**Discussion by sicusa** [share121/inter-knot#3352] [其他]
![4d6ee861e342d15cb7dee989fc8b7ccc](https://github.com/user-attachments/assets/5e01d1cd-6f03-44f0-8d43-dad86f55b778)
出生就是21级，我是天生传奇绳匠圣体口牙（）

**Comment by lanximuli** [comment]
升级要怎么做，多回复，多发帖吗

**Reply by sicusa** [reply]
目前好像是按照github里代码仓库的数量来计算的，但其实有点不太合理，仓库可以建空的或者fork，可能考虑 commits, pull requests 和 stars 的数量会更加合理一点

当然也期待绳网有自己的等级系统www

**Comment by aynami231** [comment]
🤩

**Comment by x97-1** [comment]
天生绳斤圣体<img src="https://img-static.mihoyo.com/communityweb/upload/460474087917ad69a4ff975a3132a0df.png" alt="妮可 哈哈" width="75" >
